apps/website/src/pages/stream/overlay.tsx

```diff
@@ -14,6 +14,17 @@ import logoImage from "@/assets/logo.png";
 
 import { type WeatherResponse } from "../api/stream/weather";
 
+const truncate = (value: string, max: number) => {
+  if (value.length <= max) return value;
+
+  for (const pattern of [" - ", " | ", ": "]) {
+    const index = value.indexOf(pattern);
+    if (index !== -1 && index < max) return value.slice(0, index);
+  }
+
+  return value.slice(0, max - 3) + "...";
```

Comment:
not really important but maybe use an actual ellipsis char here?
```suggestion
  return value.slice(0, max - 3) + "…";
```

Reply:
okay its like pretty much no difference in how its rendered. so dont bother

Reply:
I imagine the font we're using has a ligature for `...` to be rendered as an ellipsis anyway yeah.